html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Eduroam.pm

```diff
@@ -0,0 +1,95 @@
+package pfappserver::Form::Config::Authentication::Source::Eduroam;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Eduroam
+
+=cut
+
+=head1 DESCRIPTION
+
+Form definition to create or update an Eduroam authentication source.
+
+=cut
+
+use strict;
+use warnings;
+
+use HTML::FormHandler::Moose;
+
+extends 'pfappserver::Form::Config::Authentication::Source';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field 'tlrs1_server_address' => (
```

Comment:
You know, I am not so sure calling them tlrs1 and tlrs2 is a great idea.
Those are the current North-American servers names, but they will not make sense elsewhere.
Could we not just have server1 and server2?
Or something that would make sense globally?